Sources/SmileID/Classes/SelfieCapture/SelfieViewModel.swift

```diff
@@ -342,6 +343,7 @@ public class SelfieViewModel: ObservableObject, ARKitSmileDelegate {
         livenessImages = []
         shouldAnalyzeImages = true
         cleanUpSelfieCapture()
+        selfieCaptureRetries += 1
```

Comment:
We need to increase the retry count also here https://github.com/smileidentity/ios/blob/6ebc2c2dc5da4f44df4c49670065b7d1d6b24223/Sources/SmileID/Classes/SelfieCapture/SelfieViewModel.swift#L365